javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -2,6 +2,7 @@
 
 import com.sun.tools.javac.code.Flags;
 import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.code.Symbol.ClassSymbol;
```

Comment:
This file so far has used `ClassSymbol` without this import, so those existing uses are now inconsistent with the new ones. Either change the existing `Symbol.ClassSymbol` or don't add this import.